client-side/src/stories/popup/popup.jsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { Unstable_Popup as BasePopup } from '@mui/base/Unstable_Popup';
+import GenericButton from '../Button/GenericButton';
+import './popup.scss';
+
+export default function SimplePopup({message='',labelButtonOutThePopup='open message',iconInButton,labelButtonInThePopup='delete message',onClick}) {
+  const [anchor, setAnchor] = React.useState(null);
+
+  const handleClick = (event) => {
+    setAnchor(anchor ? null : event.currentTarget);
+  };
+
+  const open = Boolean(anchor);
+  const id = open ? 'simple-popup' : undefined;
+
+  return (
+    <div>
+      <GenericButton className="secondary" label={labelButtonOutThePopup} aria-describedby={id} type="button" onClick={handleClick} />
```

Comment:
the popup component is responsible only for the popup itself, please remove the button